Sources/Classes/core/View/MiniAppView+Definitions.swift

```diff
@@ -29,7 +29,7 @@ public struct MiniAppConfig {
     - Parameters:
         - config: MiniAppSdkConfig that defines the baseUrl and other basic settings
         - adsDisplayer: Ads Displayer for showing ads
-        - messageDelegate: Message delegate to handle getUniqueId etc
+        - messageDelegate: Message delegate to handle getMauid etc
```

Comment:
Thanks for noticing. This should be changed to something more generic, because we have lot of interfaces in `MiniAppMessageDelegate`

Reply:
See if this below sentence holds good: 
`Message delegate to handle different interface callbacks, used by Mini App to communicate with the Native implementation.`